package.json

```diff
@@ -819,6 +819,25 @@
           "default": false,
           "description": "%config.editor_prefer_editor%"
         },
+        "i18n-ally.keygenStrategy": {
+          "type": "string",
+          "default": "default",
+          "enum": [
+            "default",
```

Comment:
`slug` would be better instead of `default` I think

Reply:
done